internal/lvmd/command/lvm_command_test.go

```diff
@@ -192,5 +192,39 @@ func Test_lvm_command(t *testing.T) {
 				t.Fatalf("expected error to be %v, got %v", ErrNoMultipleOfSectorSize, err)
 			}
 		})
+
+		t.Run("create cached volume and read it", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("create cached volume and it should not classified as thin volume.", func(t *testing.T) {
```
I want to make sure that the test's purpose is clear.